entrypoint.sh

```diff
@@ -7,13 +7,14 @@ export VAULT_SKIP_VERIFY=true
 export CONSUL_HTTP_ADDR=$CONSUL_ADDR:8500
 export SERVICE_ACCOUNT_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 
-export VAULT_PATH=auth/kubernetes/login
-if [ $KUBERNETES_SERVICE_HOST == '172.20.0.1' ] || [ $KUBERNETES_SERVICE_HOST == '10.100.0.1' ]
-then
-  export VAULT_PATH=auth/eks/login
-fi
+export VAULT_PATH=auth/eks/login
```

Comment:
Does this need to be `auth/eks/`?

Reply:
dig resolves the CNAME with a . in last. The sed adds the `/` in the end due to this. We ca have a regex fix for this, but I thought this is simpler.

`vault-path.infra.svc.cluster.local. 30 IN CNAME	auth.eks.`

the CNAME after sed becomes

`/auth/eks/` to which we can add login. The default path would end in `/`

Reply:
we can remove this variable since we are updating this without using it.